ethcore/res/ethereum/ropsten.json

```diff
@@ -62,8 +62,8 @@
 		"gasLimit": "0x1000000"
 	},
 	"hardcodedSync": {
-		"header": "f90218a00379bed5cf7fb318965ec9b496aadc573fb78d1570ad2c43703c72e6c6ed997aa0b86c16b5c12556b0ea79b4d52ec77e3a3362ef43eeea51b594afc53c4b0ac115942450ec589db090d88e8de36c1790ca309841181da06183029c4e9e730abd17f5ea5427b626ed83796cab9bb4f28444cb7ab159f8f2a00ef00ef22f20f7d802cbede6ed8ff42d62671a7c07af9496e14710d7e63bbb9ea0a0e273f9d56a2ad827fb338fc0a109c28c0eee076d01454b76d42c99f8c54168b90100000c0002000000000000000000000400000000000000000000002000080000000008800010000000000000000000000002100000000000000000000020000000000000000000420000000008002000000000000000000000800001100000200004000000000000000000000008000000000000000000000000020010000000000000010000000010000040000000000000000020000000000000000028000018000000000000000000002000008000000800000000004000000000000000000000000002000009000000000810000000000040000000000000200100000000040000000000000000000200200000000000000000000000000000000000200000844af3d7d883486001837a120083146f76845c2c29239ad983010814846765746888676f312e31312e348664617277696ea071764589dfd7ecd322aee118da27aa747da4f5d14c1a5951f9b3a297d47a031888ba12cdb8daaeae57",
-		"totalDifficulty": "17277173430188370",
+		"header": "f90213a002856c991cee4843dd0055f843dba5304a2de90db9ef16f6e06970984a147fffa01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d4934794d9ee4d08a86b430544254ff95e32aa6fcc1d3163a08631a00947e31a46b0e6eadafa5a73ff6f29e9c7088f5e440206abe7687116f6a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421b9010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000832637e5834cb801837a120080845c6ab20199d883010815846765746888676f312e31302e34856c696e7578a04b5cc1554f1c30864b83b6a5bf6cf12e3b7a6c1d1150814fa0f2ac1758cffd73883701bbd33b5769fe",
+		"totalDifficulty": "18208711110298778",
```

Comment:
That's the wrong chain @Tbaut 

```
2019-02-27 12:33:43  Starting Parity-Ethereum/v2.2.11-stable-8e31051c43-20190220/x86_64-linux-gnu/rustc1.32.0
2019-02-27 12:33:43  Keys path /home/user/.local/share/io.parity.ethereum/keys/ropsten
2019-02-27 12:33:43  DB path /home/user/.local/share/io.parity.ethereum/chains_light/ropsten/db/ae90623718e47d66
2019-02-27 12:33:43  Running in experimental Light Client mode.
2019-02-27 12:33:43  Inserting hardcoded block #5027841 in chain
2019-02-27 12:33:43  Debug API is not available in light client mode.
2019-02-27 12:33:43  Debug API is not available in light client mode.
2019-02-27 12:33:43  Public node URL: enode://e2ea472bcbf28c3a5784e34f6ca05cca9c87779885af013eaf3ecc4d0f213e19a20f4170b83945631507b84187368df51dcb15011da4f2c125e9a1e58f258934@192.168.0.196:30303
2019-02-27 12:33:48  Syncing #5027841 0x9a38…9efb     0.0 hdr/s      0+    0 Qed   2/50 peers   560 bytes cache 0 bytes queue  RPC:  0 conn,    0 req/s,    0 µs
2019-02-27 12:33:53  Syncing #5027841 0x9a38…9efb     0.0 hdr/s      0+    0 Qed   1/50 peers   560 bytes cache 0 bytes queue  RPC:  0 conn,    0 req/s,    0 µs
2019-02-27 12:33:58  Syncing #5027841 0x9a38…9efb     0.0 hdr/s      0+    0 Qed   1/50 peers   560 bytes cache 0 bytes queue  RPC:  0 conn,    0 req/s,    0 µs
```

But according to etherscan it's `0xecff91fce98c94ebed5490b1b27a45c84f5a29fbd296951c7b492b391a43b3db`

https://ropsten.etherscan.io/block/5027841

Reply:
I've got a relatively up to date Ropsten node, I can update this.

Reply:
Nice one, I forgot to check that one sorry about that.